types/fs-extra/index.d.ts

```diff
@@ -1,85 +1,256 @@
-export declare function outputJSONSync(file: string, data: any): void;
-
-export declare function readJson(file: string, callback: (err: Error, jsonObject: any) => void): void;
-export declare function readJson(file: string, options: ReadOptions, callback: (err: Error, jsonObject: any) => void): void;
-export declare function readJSON(file: string, callback: (err: Error, jsonObject: any) => void): void;
-export declare function readJSON(file: string, options: ReadOptions, callback: (err: Error, jsonObject: any) => void): void;
-
-export declare function readJsonSync(file: string, options?: ReadOptions): any;
-export declare function readJSONSync(file: string, options?: ReadOptions): any;
-
-export declare function remove(dir: string, callback?: (err: Error) => void): void;
-export declare function removeSync(dir: string): void;
+export function copy(src: string, dest: string, options?: CopyOptions): Promise<void>;
+export function copy(src: string, dest: string, callback: (err: Error) => void): void;
+export function copy(src: string, dest: string, options: CopyOptions, callback: (err: Error) => void): void;
+export function copySync(src: string, dest: string, options?: CopyOptions): void;
+
+export function move(src: string, dest: string, options?: MoveOptions): Promise<void>;
+export function move(src: string, dest: string, callback: (err: Error) => void): void;
+export function move(src: string, dest: string, options: MoveOptions, callback: (err: Error) => void): void;
+export function moveSync(src: string, dest: string, options?: MoveOptions): void;
+
+export function createFile(file: string): Promise<void>;
+export function createFile(file: string, callback: (err: Error) => void): void;
+export function createFileSync(file: string): void;
+
+export function mkdirs(dir: string, options?: MkdirOptions): Promise<void>;
+export function mkdirs(dir: string, callback: (err: Error) => void): void;
+export function mkdirs(dir: string, options: MkdirOptions, callback: (err: Error) => void): void;
+export function mkdirp(dir: string, options?: MkdirOptions): Promise<void>;
+export function mkdirp(dir: string, callback: (err: Error) => void): void;
+export function mkdirp(dir: string, options: MkdirOptions, callback: (err: Error) => void): void;
+export function mkdirsSync(dir: string, options?: MkdirOptions): void;
+export function mkdirpSync(dir: string, options?: MkdirOptions): void;
+
+export function outputFile(file: string, data: any): Promise<void>;
+export function outputFile(file: string, data: any, callback: (err: Error) => void): void;
+export function outputFileSync(file: string, data: any): void;
+
+export function outputJSON(file: string, data: any): Promise<void>;
+export function outputJSON(file: string, data: any, callback: (err: Error) => void): void;
+export function outputJson(file: string, data: any): Promise<void>;
+export function outputJson(file: string, data: any, callback: (err: Error) => void): void;
```

Comment:
`outputJson` & `outputJSON` can optionally accept the same options as `writeJson`.

https://github.com/jprichardson/node-fs-extra/blob/master/docs/outputJson.md